src/injected.js

```diff
@@ -66,13 +66,16 @@ ghImport('jquery').then(function($) {
 		}
 	
 		var createListeners = function() {
-			// This handles partial updates
+			
+			// proxy replaceWith
 			var proxied = $.fn.replaceWith;
+
 			$.fn.replaceWith = function(contents) {
+
 				applyExtension(contents);
-				return proxied.apply(this, arguments);
-			};
-			
+				proxied.apply(this, arguments);

```

Comment:
Why did you remove the return? Maybe I was being overly cautious when I added it, but I don't think it hurt anything. Is the replaceWith stuff still used?